### ISSUE TITLE
Option to switch from using F-row / F-keys to regular number row

### ISSUE THREAD

**Issue by tgarder** [blocktronics/moebius#218]
I use a lot of smaller form keyboards sans f-rows and it just struck me -- I'll have a hard time using Moebius! Can't seem to find anything about it here so trying posting it and see if there's any interest. 

Problem: Keyboards without F-keys makes no happy ANSI times

Solution: Be able to switch to using number row + two following keys before backspace instead of f1-f12. Even people who prefer to use their f-row as media buttons could benefit from it, as long as they don't want to type numbers at the same time as they want to draw blocks (or type on actual buttons instead of the touch bar on macbooks)

It would of course be nice to have as an option so if you ever need to switch to typing numbers on the fly, but I'd be even OK with it as a start up parameter.

Anyway, thanks for a great piece of software!

**Comment by tracker1** [comment]
Maybe just CTRL+NUMBER keys?

**Comment by pandrr** [comment]
yeah, that is also my problem with many ansi drawing tools, also for macbooks without physical f-keys....
so i made it work: https://github.com/blocktronics/moebius/pull/220

cheers

**Comment by tgarder** [comment]
> Maybe just CTRL+NUMBER keys?

I've got about five keyboards and none of them have numpads :D It's good that the numpad feature is there but considering how I draw it sounds like it would be hard to switch between arrow keys <-> ctrl numpad fast enough to feel effective other for small adjustments.

**Comment by tgarder** [comment]
> yeah, that is also my problem with many ansi drawing tools, also for macbooks without physical f-keys.... so i made it work: #220
> 
> cheers

That's amazing!
